thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1449,7 +1449,10 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
     {
         try
         {
-            startInput(0);
+            {
+                LookAheadTimer t(slaveTimerStats, timeActivities);
```

Comment:
this isn't necessarily look ahead time, startLeftInput() can be called synchronously:
```
        if (rhsStartedBefore && isRhsConstant()) // if this is the 2nd+ iteration and the RHS is constant, don't bother restarting right, it will not be used.
        {
            startLeftInput();
        }
        else
        {
            CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInput, this));
```